src/main/groovy/nextflow/executor/AbstractGridExecutor.groovy

```diff
@@ -345,9 +345,10 @@ abstract class AbstractGridExecutor extends Executor {
             return true
         }
 
-        if( !status.containsKey(jobId) ) {
+        if( !status.containsKey(jobId) ) { // in case the status of the job was not found, we should assume it is still running 
+                                           // (see https://github.com/nextflow-io/nextflow/issues/927)
             log.trace "[${name.toUpperCase()}] queue ${queue?"($queue) ":''}status > map does not contain jobId: `$jobId`"
-            return false
+            return true
         }
```

Comment:
I don't agree on this change.

Reply:
Reason?

Reply:
We are talking about that the batch scheduler you are using return inconsistent job status, therefore the flaw in the LFS installation you are using. Also LSF is a commercial product for which you are supposed to pay a commercial assistance. I don't why NF should take care of a bug in your system.

Reply:
If you want nextflow to be successful and widely used, unfortunately it is necessary to take care of such edge cases. Plus, my main argument is that this is about the philosophy or approach that nextflow is taking here - instead of naively expecting that everything in the environment works as it should (which is obviously often not the case), it should take the approach that the environment is buggy and take the most conservative approach to recover from any possible failures. IMHO it would be a lot more useful workflow engine that way.